ipaclient/install/timeconf.py

```diff
@@ -39,6 +40,28 @@ def __backup_config(path, fstore=None):
         shutil.copy(path, "%s.ipasave" % (path))
 
 
+def get_time_source(options):
+    """
+    While in interactive installation user has to specify NTP server or pool
+    to be used in chrony configuration. This method asks user input on these
+    values in case that they were not specified before installation start.
+    """
+    if ipautil.user_input("Do you want to configure chrony "
```

Comment:
You aren't using i18n marker here. Is there no translation support for the installer?

Reply:
oh, right